dask/dataframe/io/tests/test_parquet.py

```diff
@@ -2731,6 +2731,20 @@ def test_optimize_and_not(tmpdir):
         assert_eq(a, b)
 
 
+@pytest.mark.parametrize("dtype", ["str", "bool", "float"])
+@write_read_engines()
+def test_chunksize_empty(tmpdir, write_engine, read_engine, dtype):
+    df = pd.DataFrame(
+        {
+            "a": pd.Series(dtype="int"),
+            "b": pd.Series(dtype=dtype)
+        }
+    )
+    ddf = dd.from_pandas(df, npartitions=1)
+    ddf.to_parquet(tmpdir, engine=write_engine)
+    dd.read_parquet(tmpdir, engine=read_engine, chunksize="1MiB")
```

Comment:
This is nice because it shows that this `dd.read_parquet` now succeeds, while it would fail before. We might also try going one small step further and assert that the DataFrame we get from the `read_parquet` call is the same as the one we started with before the `to_parquet` call. Specifically, we might replace the current `dd.read_parquet(tmpdir, ...)` line with something like:

```python
ddf2 = dd.read_parquet(tmpdir, ...)
assert_eq(ddf, ddf2)
```

Reply:
oh great! I was wondering if an assertion like this would be helpful, I'll add it in